requirements-test.txt

```diff
@@ -1,8 +1,7 @@
+-r requirements.txt
 mock
-vcrpy==1.10.3
-# FIXME remove the constraint after resolving
-# https://github.com/pytest-dev/pytest/issues/2966
-pytest<3.3.0
-pytest-cov<2.6.0
-pytest-catchlog
-responses==0.10.6
+vcrpy
+pytest
+pytest-cov
+responses
+tox
```

Comment:
It probably makes sense to remove this file and move most of its contents to `tox.ini` (minus `tox` itself, whose installation should be covered in the README).